webhooks/middleware/protocols/soap.py

```diff
@@ -16,6 +16,9 @@ def forward_to_SOAP_service(json_data, image):
     timestamp = json_data["data"]["timestamp_local"]
     plate = json_data["data"]["results"][0]["plate"]
     score = json_data["data"]["results"][0]["score"]
```

Comment:
```suggestion
    score = json_data["data"]["results"][0].get("score")
```

In vehicle mode the score is not sent on this key, I'm doing another integration and I noticed this error.